### PR TITLE
Switch human-readable times to UTC.

### PR DESCRIPTION
**Pull request by juliusv**
This fixes https://github.com/prometheus/prometheus/issues/829

### CODE REVIEW

**Comment by juliusv** [comment]
@fabxc @brian-brazil

File: template/template.go [hunk 1]
```diff
@@ -242,7 +242,7 @@ func NewTemplateExpander(text string, name string, data interface{}, timestamp c
 				if math.IsNaN(v) || math.IsInf(v, 0) {
 					return fmt.Sprintf("%.4g", v)
 				}
-				t := clientmodel.TimestampFromUnixNano(int64(v * 1000000000)).Time()
```
Comment (fabxc) [inline template/template.go]: Can we make this `1e9` for clarity?

**Comment by fabxc** [comment]
:+1: Otherwise

File: template/template.go (hunk 1 above)
Comment (juliusv) [inline template/template.go]: Thanks, done.

**Pull request merged**